risc0/zkvm/platform/src/syscall.rs

```diff
@@ -895,8 +895,7 @@ pub extern "C" fn sys_exit(status: i32) -> ! {
 /// `control_id` must be aligned and dereferenceable.
 ///
 /// `input` must be aligned and have `input_len` u32s dereferenceable
-#[cfg(feature = "export-syscalls")]
-#[no_mangle]
+#[cfg_attr(all(feature = "export-syscalls", feature = "unstable"), no_mangle)]
```

Comment:
Could we just drop the `export-syscall` feature? I'm not sure what it's for.

Reply:
We certainly could. The only reason I see for it is to give a way to prevent those symbols from ending up in the build, in case that were to cause some issues e.g. if another package implements them.